project1/application.py

```diff
@@ -37,7 +46,13 @@ def register():
         uname = request.form.get("uname")
         pwd = request.form.get("pwd")
         email = request.form.get("email")
-        print(uname+"    "+pwd + " "+email)
-        names[email] = [uname, pwd]
-        return render_template("sucess.html", mail=email, username=uname, name=names)
+        print(uname+" "+pwd + " "+email)
+        check = user.query.filter_by(email=email).first()
+        if check is None:
+            register = user(username=uname, email=email, password=pwd)
+            db.session.add(register)
+            db.session.commit()
+        else:
+            return "Failed email already registered"
+        return render_template("sucess.html", mail=register.email, username=register.username, register=register)
```

Comment:
Good job on checking if the user is already registered. Also,

- add a check to see if user submitted empty fields. 
- returning text is not a good practice. Use html pages. You can leverage "success.html" to show error messages as well.